src/main/java/com/google/devtools/build/lib/profiler/memory/AllocationTrackerModule.java

```diff
@@ -34,7 +34,12 @@
  *
  * <ol>
  *   <li>--host_jvm_args=-javaagent:(path to Google's java agent jar)
- *   <li>--host_jvm_args=-RULE_MEMORY_TRACKER=1
+ *     <ul>
+ *       <li>For Bazel use 
+ *         <a href="https://github.com/bazelbuild/bazel/tree/master/third_party/allocation_instrumenter">java-allocation-instrumenter-3.0.1.jar</a>
+ *       </li>
+ *    </ul>
+ *   <li>--host_jvm_args=-DRULE_MEMORY_TRACKER=1
```

Comment:
Forgot to mention that the "D" was missing here which is crucial to getting things working, so even if we don't want the update regarding where the jar is, we need to fix this bit.